### PR TITLE
created signup, login and forgot password page

### PR DESCRIPTION
**Pull request by kgajriaAgarwal**

### CODE REVIEW

File: Authentication/ForgotPassword.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+                alt="login-img" />
+        </div>
+        <div class="col-7 login-content-container">
+            <div class="login-form">
+                <img src="../Assets/Blithe-logo.png" class="blithe-logo" alt="blithe-logo"/>
+                <form>
+                    <p class="heading-md heading">Forgot Password</p>
+                    <div class="float-container">
+                        <label for="floatField" class="input-lbl">Email</label>
+                        <input type="email" name="">
```
Comment (git-0r) [inline Authentication/ForgotPassword.html]: you can use 'required' attribute for input validation before form submit

**Review by git-0r — APPROVED** [review]

**Review by JuvitaSaini09 — APPROVED** [review]
everything looks good to me

File: Authentication/ForgotPassword.html (hunk 1 above)
Comment (kgajriaAgarwal) [reply Authentication/ForgotPassword.html]: ok

**Comment by kgajriaAgarwal** [comment]
changes according to pr review

**Pull request closed without merging**